### PR TITLE
[Rule Tuning] Update EQL rules with lookback < maxspan

### PR DESCRIPTION
**Pull request by brokensound77**
## Issues
related to #1343 
related to #1361 

## Summary
Updates EQL rules which had a longer maxspan defined than lookback

### CODE REVIEW

File: rules/windows/defense_evasion_mshta_beacon.toml [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
+updated_date = "2021/07/19"
 
 [rule]
 author = ["Elastic"]
 description = """
 Identifies Mshta.exe making outbound network connections. This may indicate adversarial activity as Mshta is often
 leveraged by adversaries to execute malicious scripts and evade detection.
 """
-from = "now-9m"
+from = "now-120m"
```
Comment (rw-access) [inline rules/windows/defense_evasion_mshta_beacon.toml:12]: we should adjust the interval as well.
what's our default interval?

i think a healthy metric is for the interval to be >= 1/2 maxspan

Comment (rw-access) [reply rules/windows/defense_evasion_mshta_beacon.toml:12]: since these are process+network, i would say we just decrease maxspan to 10m

Comment (brokensound77) [reply rules/windows/defense_evasion_mshta_beacon.toml:12]: default is `5m`. 

Nice suggestion, updating

Comment (brokensound77) [reply rules/windows/defense_evasion_mshta_beacon.toml]: 319e6db

test added for intervals in #1361

Comment (brokensound77) [reply rules/windows/defense_evasion_mshta_beacon.toml]:
```
E           AssertionError: The following rules have longer max_spans than lookbacks:
E           c2d90150-0133-451c-a783-533e736c12d7 - Mshta Making Network Connections -> lookback: 540000, maxspan: 7200000
E           f036953a-4615-4707-a1ca-dc53bf69dcd5 - Unusual Child Processes of RunDLL32 -> lookback: 540000, maxspan: 3600000
E           0d8ad79f-9025-45d8-80c1-4f0cd3c5e8e5 - Execution of File Written or Modified by Microsoft Office -> lookback: 540000, maxspan: 7200000
E           1defdd62-cd8d-426e-a246-81a37751bb2b - Execution of File Written or Modified by PDF Reader -> lookback: 540000, maxspan: 7200000
```

File: rules/windows/defense_evasion_mshta_beacon.toml [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
+updated_date = "2021/07/19"
 
 [rule]
 author = ["Elastic"]
 description = """
 Identifies Mshta.exe making outbound network connections. This may indicate adversarial activity as Mshta is often
 leveraged by adversaries to execute malicious scripts and evade detection.
 """
-from = "now-9m"
+from = "now-10m"
```
Comment (rw-access) [inline rules/windows/defense_evasion_mshta_beacon.toml:12]: I think we need more margin to account for ingest delays. Having the interval match the sequence max span exactly is tricky

Comment (rw-access) [reply rules/windows/defense_evasion_mshta_beacon.toml:12]:
```suggestion
from = "now-20m"
```

**Review by rw-access — APPROVED** [review]
one fix then LGTM

File: rules/windows/defense_evasion_mshta_beacon.toml (hunk 2 above)
Comment (brokensound77) [reply rules/windows/defense_evasion_mshta_beacon.toml]: Also updated tests to give 10m buffer baf23b503f9d3004c2f0782ad1a95c7f7eed8c13

**Pull request merged**